sympy/core/numbers.py

```diff
@@ -1370,7 +1389,7 @@ def __format__(self, format_spec):
 RealNumber = Float
 
 
-class Rational(Number):
+class Rational(with_metaclass(ManagedABCMetaclass, Number, numbers.Rational)):
```

Comment:
```suggestion
class Rational(with_metaclass(ManagedABCMetaclass, Number, num.Rational)):
```
And here

Reply:
@Sc0rpi0n101 those changes didn't work. Not sure what's causing the conflict. Any other ideas?